src/test/java/models/testAmericanGame.java

```diff
@@ -4,43 +4,50 @@
 
 import static org.junit.Assert.*;
 
+import static org.hamcrest.CoreMatchers.instanceOf;
+
 /**
- * Created by michaelhilton on 1/25/16.
+ * Created by ramcharan on 2/19/16.
  */
-public class testGame {
-/*
+public class testAmericanGame
+{
     @Test
-    public void testGameCreation(){
-        Game g = new Game();
+    public void testAmGameCreation()
+    {
+        Game g = new amGame();
         assertNotNull(g);
     }
 
     @Test
-    public void testGameBuildDeck(){
-        Game g = new Game();
-        g.buildDeck();
-        assertEquals(52,g.deck.size());
+    public void testAmGameClass()
+    {
+        Game g = new amGame();
+        assertThat(g,instanceOf(amGame.class));
     }
 
     @Test
-    public void testSpanishGameBuildDeck()
+    public void testBuildDeck()
     {
-        Game g = new Game();
-        g.buildSpanishDeck();
-        assertEquals(40,g.deck.size());
+        Game g = new amGame();
+        g.buildDeck();
+
+        assertEquals(52,g.deck.size());
     }
 
     @Test
-    public void testGameInit(){
-        Game g = new Game();
+    public void testAmGameShuffle()
+    {
+        Game g = new amGame();
         g.buildDeck();
         g.shuffle();
+
         assertNotEquals(2,g.deck.get(0).getValue());
     }
 
     @Test
-    public void testGameStart(){
-        Game g = new Game();
+    public void testAmGameStart()
```

Comment:
LIttle confused by this test because there is no assert so I am not sure at what is being tested.